ci-operator/config/openshift/cloud-provider-gcp/openshift-cloud-provider-gcp-master.yaml

```diff
@@ -100,6 +104,19 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp-upgrade
+- as: regression-clusterinfra-cucushift-rehearse-gcp-ipi
+  optional: true
+  steps:
+    cluster_profile: gcp-3
+    env:
+      TEST_FILTERS: NonHyperShiftHOST
+      TEST_SCENARIOS: Cluster_Infrastructure
```

Comment:
we should set TEST_SCENARIOS with `Cluster_Infrastructure CCM`?

Reply:
Thanks @sunzhaohua2 , just realized was using the earlier way , which isn't efficient .